TutorialApplication/MainViewController.m

```diff
@@ -64,21 +73,23 @@ - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
 {
-    return [self.organization.employees count];
+    return self.organization.employees.count;
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
 {
     UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"EmployeeCell" forIndexPath:indexPath];
     
-    cell.textLabel.text = self.organization.employees[indexPath.row].firstName;
+    FFEmployee *employee = self.organization.employees.allObjects[indexPath.row];
+    
+    cell.textLabel.text =employee.fullName;
     
     return cell;
 }
 
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
-    self.selectedEmployee = self.organization.employees[indexPath.row];
+    self.selectedEmployee = self.organization.employees.allObjects[indexPath.row];
```

Comment:
.allObject does not guarantee order of the objects.
You should add a read-only property to organization class NSArray<Employee *> *sortedEmployees, which should dynamically sort self.employees.